ydb/core/fq/libs/config/protos/compute.proto

```diff
@@ -17,10 +17,22 @@ message TSynchronizationService {
 message TInPlaceCompute {
 }
 
+message TLoadControlConfig {
+    bool Enable = 1;
+    string MonitoringRequestDelay = 2;     // default 0 == non-stop polling
```

Comment:
As i understand this call is heavy and we need to call it as little as possible. Is it possible to change the default option to several seconds so that there is no mistake during the configuration

Reply:
Changed to "1s"